.changeset/little-roses-kick.md

```diff
@@ -0,0 +1,14 @@
+---
+"@jspsych/plugin-canvas-keyboard-response": patch
+"@jspsych/plugin-image-keyboard-response": patch
```

Comment:
I wouldn't consider a package's tests "public API", so no need to release anything but the config package IMHO.